### ISSUE TITLE
parser fails to load on Ubuntu 20.04 (Python3.8)

### ISSUE THREAD

**Issue by tnagano22** [nagataaaas/Japanera#6]
Parser failed to load on Ubuntu 20.04 (Python3.8).

- System information
     - Ubuntu version        : 	Ubuntu 20.04.6 LTS
    - Processor architecture: aarch64
    - CPU information       : ARM 32-bit, 64-bit
    - Python3 version       : Python 3.8.10

```
Python 3.8.10 (default, May 26 2023, 14:05:08) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import japanera
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/tnagano/.local/lib/python3.8/site-packages/japanera/__init__.py", line 17, in <module>
    from .japanera import (Era, EraDate, EraDateTime, ERA_DATA_COMMON, ERA_DATA_DAIKAKUJI, ERA_DATA_JIMYOUIN,
  File "/home/tnagano/.local/lib/python3.8/site-packages/japanera/japanera.py", line 10, in <module>
    from .parser import _strptime, find_era_and_date, _set_era_data
  File "/home/tnagano/.local/lib/python3.8/site-packages/japanera/parser.py", line 536, in <module>
    def find_eras_with_year(year: int) -> set["Era"]:
TypeError: 'type' object is not subscriptable
```

**Comment by nagataaaas** [comment]
Sorry I was not aware that this issue had been created.
I will fix this behavior in the next version (2.10).

**Comment by nagataaaas** [comment]
2.1.0 has been released
